src/openrct2/ride/RideData.cpp

```diff
@@ -356,3 +355,16 @@ ResearchCategory RideTypeDescriptor::GetResearchCategory() const
     log_error("Cannot get Research Category of invalid RideCategory");
     return ResearchCategory::Transport;
 }
+
+// used by _legacy.cpp and RideConstruction.cpp
```

Comment:
```suggestion
// used by _legacy.cpp and RideConstruction.cpp
```
static can be only used in a single TU, drop comment.